### PR TITLE
amam/lighthouse accessibility and performance issue fix

### PR DESCRIPTION
**Pull request by mustofa-binary**
- improve perf
- improve acc

### CODE REVIEW

**Comment by ghost** [comment]
Please resolve conflicts

File: src/components/elements/input/input.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import { InputStyle, ErrorText, Label } from './input-style'
 import { VFlex } from 'components/elements'
 
-export const Input = ({ error, className, name, ...props }) => {
+export const Input = ({ error, className, id, ...props }) => {
     return (
         <VFlex className={className}>
-            <Label htmlFor="email">{name}</Label>
```
Comment (ghost) [inline src/components/elements/input/input.js]: This adds an email label. in the design there is no label is that fine?

**Review by ghost — DISMISSED** [review]

File: src/components/elements/input/input.js (hunk 1 above)
Comment (mustofa-binary) [reply src/components/elements/input/input.js]: ive added display: none to the label, so it doesnt show in the UI. label is necessary to improve accessibility score

File: src/components/page-elements/nav/nav.js [hunk 2]
```diff
@@ -5,32 +5,32 @@ import { localize } from 'components/localization'
...
+    //     document.addEventListener('scroll', handleScroll, { passive: true })
 
-    return () => {
-      document.removeEventListener('scroll', handleScroll)
-    }
-  }, [])
+    //     return () => {
+    //       document.removeEventListener('scroll', handleScroll)
+    //     }
+    //   }, [])
```
Comment (ghost) [inline src/components/page-elements/nav/nav.js]: Do we need these commented lines later or they can be removed?

**Review by ghost — APPROVED** [review]

**Pull request merged**